admin/server/api/list/download.js

```diff
@@ -11,10 +12,10 @@ module.exports = function (req, res) {
 		catch (e) { } // eslint-disable-line no-empty
 	}
 	if (typeof filters === 'object') {
-		req.list.addFiltersToQuery(filters, where);
+		where = assign({}, req.list.addFiltersToQuery(filters));
 	}
 	if (req.query.search) {
-		req.list.addSearchToQuery(req.query.search, where);
+		where = assign({}, req.list.addSearchToQuery(req.query.search));
```

Comment:
It's a valid use case to provide _both filters and a search_, so these lines should be

```
assign(where, req.list.addSearchToQuery(req.query.search));
```